pkg/cmd/node/node_diagnostics_status.go

```diff
@@ -0,0 +1,43 @@
+package node
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+func newCmdNodeDiagnosticsStatus(ctx api.Context) *cobra.Command {
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Print diagnostics job status",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			status, err := diagnosticsClient().Status()
+			if err != nil {
+				return err
+			}
+
+			if jsonOutput {
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(status)
+			}
+			for k, v := range status {
+				fmt.Println(k)
+				val := reflect.ValueOf(v)
+				t := val.Type()
+				for i := 0; i < t.NumField(); i++ {
+					fmt.Printf("  %s: ", t.Field(i).Tag.Get("json"))
```

Comment:
For the record, this would print unwanted data in case there are options like `omitempty` on the json tag. I think it'd be more idiomatic Go to add a `String()` method to the `BundleReportStatus`.

Reply:
Regarding whats being printed here overall, yesterday I talked to @ArmandGrillet and mentioned it might be worth redesigning the output. What is the benefit of having this mixed json/text output and not a table like everywhere else?

Reply:
I think a table would be unreadable because of some fields being quite long and was probably the reason for choosing this approach. We will have a ticket about reviewing the new node subcommand and its UX on the next sprints, where we can consider changing the output.